proposals/dao/vip_14.ts

```diff
@@ -0,0 +1,214 @@
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { assertApproxEq } from '@test/helpers';
+
+/*
+
+Volt Protocol Improvement Proposal #14
+
+Description: 
+
+/// Deployment Steps
+/// 1. deploy morpho dai deposit
+/// 2. deploy morpho usdc deposit
+/// 3. deploy compound pcv router pointed to morpho dai and usdc deposits
+/// 4. deploy volt system oracle
+/// 5. deploy maple usdc deposit
+
+Governance Steps:
+1. Grant Morpho PCV Router PCV Controller Role
+2. Revoke PCV Controller Role from Compound PCV Router
+3. Remove Compound DAI Deposit from ERC20Allocator
+4. Remove Compound USDC Deposit from ERC20Allocator
+5. Add Morpho DAI Deposit to ERC20Allocator
+6. Add Morpho USDC Deposit to ERC20Allocator
+7. Add USDC and DAI Morpho deposits to the PCV Guardian
+8. Point Oracle Pass Through to new oracle address
+
+*/
+
+/// TODO update this to correct start time
+let startTime;
+
+const monthlyChangeBasisPoints = 29;
+
+const vipNumber = '14';
+
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  startTime = Math.floor(Date.now() / 1000).toString();
+
+  const voltSystemOracle = await ethers.getContractAt('VoltSystemOracle', addresses.voltSystemOracle);
+
+  const currentPrice = await voltSystemOracle.getCurrentOraclePrice();
+
+  const maplePCVDepositFactory = await ethers.getContractFactory('MaplePCVDeposit');
+  const morphoPCVDepositFactory = await ethers.getContractFactory('MorphoCompoundPCVDeposit');
+  const compoundPCVRouterFactory = await ethers.getContractFactory('CompoundPCVRouter');
+  const voltSystemOracleFactory = await ethers.getContractFactory('VoltSystemOracle');
+
+  const daiMorphoCompoundPCVDeposit = await morphoPCVDepositFactory.deploy(addresses.core, addresses.cDai);
+  await daiMorphoCompoundPCVDeposit.deployed();
+
+  const usdcMorphoCompoundPCVDeposit = await morphoPCVDepositFactory.deploy(addresses.core, addresses.cUsdc);
+  await usdcMorphoCompoundPCVDeposit.deployed();
+
+  const morphoCompoundPCVRouter = await compoundPCVRouterFactory.deploy(
+    addresses.core,
+    daiMorphoCompoundPCVDeposit.address,
+    usdcMorphoCompoundPCVDeposit.address
+  );
+  await morphoCompoundPCVRouter.deployed();
+
+  const maplePCVDeposit = await maplePCVDepositFactory.deploy(addresses.core, addresses.mplPool, addresses.mplRewards);
```

Comment:
TODO mplRewards and mplPool are placeholder values, replace with actual values

Reply:
triple checked addresses in mainnetAddresses.ts matched up with orthogonal pool on maple https://maplefinance.gitbook.io/maple/api/public/pools